src/app/components/registration-form/registration-form.component.ts

```diff
@@ -15,14 +18,19 @@ export class RegistrationFormComponent {
         password: '',
     };
```

Comment:
It's not necessary to create empty user object here.

Reply:
But it doesn't work without it... :D

Reply:
Ok. So mabey it is necessary :D. Sry then.